prime-angular-frontend/src/app/modules/site-registration/pages/remote-users-page/remote-users-page.component.html

```diff
@@ -24,6 +24,21 @@
 
   </form>
 
+  <div class="mb-4"
+       *ngIf="addedFirstRemoteUser">
```

Comment:
So this banner you just want to show when atleast 1 remote user has been added. I don't think you need any of this query param passing, you can just observe the formState. Since in this component you are already rendering cards based on remote users.

So here just check
``` *ngIf="formState.remoteUsers.length > 0" ```

Or something along those lines.

Reply:
Asking Pilar in the Jira ticket re: expected behavior

Reply:
Maybe I'm misunderstanding you @hogmann but we can't display the banner if the user hasn't actually just added a Remote Practitioner.  I think for what you're suggesting, the banner would display as long as there are any Remote Practitioners associated with the site, correct?

Reply:
So my understanding from 
``` once a remote practitioner is added show green banner “remote practitioner added, continue adding remote practitioners if this site has more than one”. ```

Is we show this banner once we have added a remote user. So we already have the code that only adds the query param "fromRemoteUser" when you have successfully submitted the form on the previous page. So with that param and the amount of remote users in the list we know if we have added/updated a remote user. So we can show the banner in that case.

Then since there is that one corner case that if the user updates a user and submits the banner will show to. So in that case it might not be unreasonable just to say on the banner remote user added/updated

Reply:
Then to handle Amir's bug with if you remove them from that page the banner is still there you can just clear whatever flag you set when you see the "fromRemoteUser" on removal of a user.

Reply:
Should be addressed now